lib/metadata/wrapper.js

```diff
@@ -67,54 +67,55 @@ const metadata = {
         });
     },
 
-    putObjectMD: (bucketName, objName, objVal, log, cb, params) => {
+    putObjectMD: (bucketName, objName, objVal, params, log, cb) => {
         log.debug('putting object in metdata');
-        client.putObject(bucketName, objName, objVal, log, err => {
+        client.putObject(bucketName, objName, objVal, params, log, err => {
             if (err) {
                 log.warn('error from metadata', { implName, error: err });
                 return cb(err);
             }
             log.debug('object successfully put in metadata');
             return cb(err);
-        }, params);
+        });
     },
 
-    getBucketAndObjectMD: (bucketName, objName, log, cb, params) => {
+    getBucketAndObjectMD: (bucketName, objName, params, log, cb) => {
         log.debug('getting bucket and object from metadata',
                   { database: bucketName, object: objName });
-        client.getBucketAndObject(bucketName, objName, log, (err, data) => {
-            if (err) {
-                log.debug('error from metadata', { implName, err });
-                return cb(err);
-            }
-            log.debug('bucket and object retrieved from metadata',
-                      { database: bucketName, object: objName });
-            return cb(err, data);
-        }, params);
+        client.getBucketAndObject(bucketName, objName, params, log,
```

Comment:
It would be cleaner to replace the anonymous callback function of `client.getBucketAndObject` by `cb` directly. Applications using metadata wrapper should catch and log the error if it happens.